pkg/unvanquished_src.dpkdir/ui/options_graphics.rml

```diff
@@ -135,13 +135,16 @@
 			<panel>
 				<h2><translate>General</translate></h2>
 				<row>
-					<select cvar="r_picmip" style="width: 20em;">
-						<option value="2"><translate>Low</translate>&nbsp;(1:4)</option>
-						<option value="1"><translate>Medium</translate>&nbsp;(1:2)</option>
-						<option value="0"><translate>Full</translate>&nbsp;(1:1)</option>
+					<select cvar="r_imageMaxDimension">
+						<option value="4096">4096&nbsp;<translate>(ultra)</translate></option>
+						<option value="2048">2048&nbsp;<translate>(high)</translate></option>
+						<option value="512">512&nbsp;<translate>(medium)</translate></option>
+						<option value="256">256&nbsp;<translate>(low)</translate></option>
+						<option value="64">64&nbsp;<translate>(lowest)</translate></option>
+						<option value="0"><translate>No limit</translate></option>
```

Comment:
ISTM this one should be sorted as higher than ultra.

Reply:
Done.